src/Supervisor.cpp

```diff
@@ -1221,4 +1221,63 @@ ZunResult Supervisor::FadeOutMusic(f32 fadeOutSeconds)
     return ZUN_SUCCESS;
 }
 #pragma optimize("", on)
+
+#pragma optimize("s", on)
```

Comment:
This function certainly isn't optimized, my process of trough its "if there is a `leave` instruction near the end, use optimize, otherwise, dont", i think that there are some functions that are optimized and dont have a `leave` instruction, but its pretty rare